### PR TITLE
PDAs get some form of writing implement by default

### PR DESCRIPTION
**Pull request by DisturbHerb**
[WIKI]

## About the PR 
PDAs, by default, will start with a pen in its pen slot. Some jobs have differing writing utensils, as follows.

- Head roles get fancy pens, fancy shmancy.
- Silicons don't get any writing utensils, they don't even have hands.
- The Detective starts with a cigarette in their PDA. Ha.
- The Clown gets a randomly coloured crayon.

## Why's this needed? 
I like pens.

## Changelog

```changelog
(u)DisturbHerb
(+)(Most) PDAs will start with writing utensils in them by default.
```

### CODE REVIEW

**Comment by Zonespace27** [comment]
You might want to remove scientists having pens in their pockets, since they'll have one in their PDAs.

File: code/obj/item/device/pda2/pda2.dm [hunk 1]
```diff
@@ -890,7 +914,7 @@
 			src.UpdateOverlays(null, "pen")
 			return
 
-	proc/insert_pen(obj/item/insertedPen, mob/user)
+	proc/insert_pen(var/obj/item/insertedPen, mob/user)
```
Comment (ZeWaka) [inline code/obj/item/device/pda2/pda2.dm:917]:
```suggestion
	proc/insert_pen(obj/item/insertedPen, mob/user)
```
nah fam

**Review by ZeWaka — APPROVED** [review]

**Comment by DisturbHerb** [comment]
> You might want to remove scientists having pens in their pockets, since they'll have one in their PDAs.

I'll leave that for when this gets merged, unless it's not too out of scope.

**Review by Gannets — APPROVED** [review]
Agree with removing the pocket pen too.

**Pull request merged**